### PR TITLE
Should fix the problems, will try and fix the history to preserve privacy as needed

### LINKED ISSUE #33: Security-- .gitignore application.properties
As a developer I don't want my private PostgreSQL passwords out and about on GitHub creating security issues. 

**Acceptance Criteria:**

- [ ]  Add application.properties to the .gitignore
- [ ]  Add a application.properties.example to the git repo to give the general format and layout

### PR DESCRIPTION
**Pull request by MCoriale**
Closes #33 

Removed application.properties from Git, added application.properties.example, both at @DRSisco 's suggestion.

### CODE REVIEW

**Review by hanseljt77 — COMMENTED** [review]
Looking forward to not needing to put my password in everytime!

File: src/main/resources/application.properties.example [hunk 1]
```diff
@@ -1,9 +1,8 @@
-
 spring.jpa.database=POSTGRESQL
 spring.datasource.platform=postgres
 spring.datasource.url=jdbc:postgresql://localhost:5432/CovidTracer
 spring.datasource.username=postgres
-spring.datasource.password=Jaeakecc7
+spring.datasource.password=YOUR-PASSWORD-HERE
```
Comment (noahthomas88) [inline src/main/resources/application.properties.example:5]: How Generic and inconspicuous! Very nice!

**Review by noahthomas88 — APPROVED** [review]
Good to go! Good Work :D

**Review by hanseljt77 — COMMENTED** [review]
Just noticed that application.properties is now named application.properties.example. Is there a reason for this change? Won't this just mean it still isn't ignored? Maybe I misunderstand how gitignores work.

**Comment by DRSisco** [comment]
@hanseljt77 https://anidiots.guide/other-guides/env-files#using-git-ex-github

Basically you git ignore the real file, then you include an example file with the list of environment variables. Typically defaults are put in the example, though you know as a dev to modify your env file to reflect those variables and what you need in your set up. 
Same is true for application.properties in spring. Though I am not sure of what you'd call it in spring, so this name of adding .example is used to match most other languages/frameworks.

**Comment by hanseljt77** [comment]
Ah, thanks for the explanation. Makes sense

**Pull request merged**